### ISSUE TITLE
Settings object has no attribute "DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP"

### ISSUE THREAD

**Issue by amodia** [learningequality/ka-lite#4113]
**Branch:** 0.14.x
**Current behavior:** Can't load videos tab to because of this error `Settings object has no attribute DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP`

**Steps to reproduce:** 
 1. Log in as admin user
 2. Navigate to videos tab.

![screen shot 2015-07-17 at 10 06 50 am](https://cloud.githubusercontent.com/assets/8663934/8740071/b6efdca2-2c76-11e5-95d2-8e5e55dbae1a.png)

**Comment by cpauya** [comment]
@amodia Here's my quick patch, so all references to `settings.DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP` will have a default value of `False`.

```diff
diff --git a/kalite/settings/__init__.py b/kalite/settings/__init__.py
index 718962a..c9db9eb 100644
--- a/kalite/settings/__init__.py
+++ b/kalite/settings/__init__.py
@@ -46,6 +46,8 @@ if CONFIG_PACKAGE:
         RemovedInKALite_v015_Warning
     )
 
+DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP = getattr(local_settings, "DO_NOT_RELOAD_CONTENT_CACHE_AT_STARTUP", False)
+
 # Config for Raspberry Pi distributed server
 if package_selected("RPi"):
 ```

**Comment by rtibbles** [comment]
Fixed in #4116
